knative-operator/pkg/controller/knativeservingobsolete/deploy

```diff
@@ -0,0 +1 @@
+../../../deploy
```

Comment:
This is not actually a file but a symlink 😂 . Github could do a better job at showing that. It's needed so that the tests can access the directory correctly to not break the orphaning logic.